### PR TITLE
[MIG] membership_prorrate_variable_period: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
Depends on:

- [x] membership_variable_period: https://github.com/OCA/vertical-association/pull/32

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rebase

File: membership_prorrate/views/product_template_views.xml [hunk 1]
```diff
@@ -10,6 +10,6 @@
                     <field name="membership_prorrate"/>
                 </field>
             </field>
-        </record>
+        </record>z
```
Comment (pedrobaeza) [inline membership_prorrate/views/product_template_views.xml]: You have added accidentally here an extra letter

**Review by pedrobaeza — APPROVED** [review]
Except the letter thing. @luismontalba please review

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Comment by chienandalu** [comment]
@pedrobaeza Rebased and all CIs green

**Pull request merged**